python/source.py

```diff
@@ -41,13 +41,13 @@ def __init__(self, frequency, start_time=0, end_time=float('inf'), width=0, cuto
 
 class GaussianSource(SourceTime):
 
-    def __init__(self, frequency, width, start_time=0, cutoff=5.0):
+    def __init__(self, frequency, width, fwidth=True, start_time=0, cutoff=5.0):
         super(GaussianSource, self).__init__()
         self.frequency = frequency
-        self.width = width
-        self.start_time = 0
+        self.width = 1 / width if fwidth else width
```

Comment:
Or alternatively it would be more consistent to use `width=0, fwidth=float('inf')` and maybe set `self.width = max(width, 1/fwidth)`